src/drag_race/include/DragRaceNode.h

```diff
@@ -0,0 +1,97 @@
+/*
+ * Created By: Gareth Ellis
+ * Created On: July 16th, 2016
+ * Description: An example node that subscribes to a topic publishing strings,
```

Comment:
Description needs updating, else i'll be impressed if we're drag racing with this

Reply:
Updated